contact.html

```diff
@@ -0,0 +1,200 @@
+      <div class="mobile-menu">
+        <div class="d-flex justify-content-between align-items-center">
+          <div class="nav-menu">
+            <a href="index.html"
+              ><img
+                class="mobile-logo"
+                src="./images/DevBootcamp-White.svg"
+                alt="Amkam DevBootcamp Logo"
+            /></a>
+          </div>
+          <svg
+            class="w-6 h-6 close-icon"
+            fill="currentColor"
+            viewBox="0 0 20 20"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              fill-rule="evenodd"
+              d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+              clip-rule="evenodd"
+            ></path>
+          </svg>
+        </div>
+
+        <div class="menu-list my-5">
+          <ul class="d-flex flex-column pe-1">
+            <li class="me-5 my-4 d-block text-center">
+              <a href="about.html" class="mobile-links">About</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="mobile-links">Courses</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="mobile-links">Join</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="mobile-links">Sponsor</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="mobile-links">News</a>
+            </li>
+          </ul>
+          <div class="d-flex justify-content-center mt-5">
+            <a href="contact.html" class="mobile-enroll-button">Join us now</a>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <!-- Contact Form -->
+    <section
+      class="contact-form d-flex justify-content-center align-items-center"
+    >
+      <div>
+        <h2 class="text-center">Contact Us</h2>
+        <div class="line"></div>
+
+        <form action="https://formspree.io/f/mayarpel" method="POST">
+          <div>
+            <input
+              type="text"
+              name="user_name"
+              placeholder="Enter full name"
+              required
+            />
+          </div>
+          <div>
+            <input
+              type="email"
+              name="user_email"
+              placeholder="Enter email address"
+              required
+            />
+          </div>
+          <div>
+            <textarea
+              name="user_message"
+              id="message"
+              cols="40"
+              rows="5"
+              maxlength="500"
+              required
+              placeholder="Write your message here..."
+            ></textarea>
+          </div>
+          <button type="submit">Send</button>
+        </form>
+      </div>
```

Comment:
**Changes Required**
- [ ] Let us please replace the immediate children of all HTML5 `section` elements in the **entire code base** with the HTML5 `article` element. The use of HTML5 semantics greatly improves our web page `accessibility`.

Reply:
Sure, I can do that boss. 👍

Reply:
Well solved